impierce/identity-wallet
# fix: Remember tab when navigating back to `/me` route

# Description of change

When navigating away from the credentials overview on `/me` route, the active tab is now stored in page state and restored when you navigate back with `history.back()`.

## Links to any relevant issues

- Fixes #269.

## How the change has been tested

Manual testing:

- Switch to a tab other than *All* and open a credential. When you navigate back, via the top navigation, the same tab should be active.
- Also test navigating to the profile and a sub-route of the profile, and then navigating back via the top navigation.
- When you navigate away from `/me` via the bottom navigation, the active tab is not preserved. This is because the navigation does not happen via `history.back()`.

## Definition of Done checklist

Add an `x` to the boxes that are relevant to your changes.
